python/cuml/preprocessing/text/stem/porter_stemmer_utils/suffix_utils.py

```diff
@@ -73,7 +73,7 @@ def replace_suffix(word_str_ser, suffix, replacement, can_replace_mask):
         return stem_ser + replacement_ser
 
 
-@cuda.jit("void(int32[:], boolean[:], int32)")
+@cuda.jit()
```

Comment:
Small question. I wonder if we should use [`autojit`]( https://numba.pydata.org/numba-doc/0.13/CUDAJit.html#numba.cuda.autojit ) in this case. WDYT?

Reply:
I have some memory that autojit was deprecated? I'm not seeing it [here](https://numba.readthedocs.io/en/stable/cuda-reference/kernel.html), but let me know if I totally imagined the deprecation.

Reply:
Aha! Yep, here's the PR that removed it: https://github.com/numba/numba/pull/5519

Reply:
Ah fair enough. Feel free to mark as resolved then :)